AmericanMedicalAssociation/ama-style-guide-2
# EWL-5147 - add purple and blue h4

## Ticket(s)

**Jira Ticket**
- [EWL-5147: SG2 | Correct pagination link color and add padding](https://issues.ama-assn.org/browse/EWL-5147)

## Description
Add h4 purple and blue versions

## To Test

- Pull `bugfix/EWL-5147-add-h4-styles` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- visit http://localhost:3000/?p=pages-topic
- observe that the lead news is now purple and has the purple h4 styling
- the purple text should be the following sizes: 
Desktop: Myriad Pro Regular 18/35
Mobile: Myriad Pro Regular 16/23
Tablet: Myriad Pro  Regular 18/28
- use Chrome to inspect the lead news text and change the class to from `ama__h4--purple` to `ama__h4--blue`
- observe the text turns blue with the following sizes:
Desktop:  Myriad Pro Regular 28/35
Mobile: Myriad Pro Regular 18/22
Tablet: Myriad Pro  Regular 18/22

## Visual Regressions
There is no VRT testing for h4 atom elements
N/A

## Relevant Screenshots/GIFs
<img width="1259" alt="screen shot 2018-05-23 at 4 21 29 pm" src="https://user-images.githubusercontent.com/2271747/40451777-5f214f38-5ea5-11e8-9906-dd1bbbc77c84.png">


## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
